example/95-audit-policy-for-shoot.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: auditpolicy
```

Comment:
In the shoot example manifests we state [`namespace: garden-dev`](https://github.com/gardener/gardener/blob/master/example/90-shoot-aws.yaml#L6), thus, to make it more clear please also state it here.